vue/components/molecules/messages-alert/messages-alert.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+    <transition

```

Comment:
This needs to use the same strategy as `restrictions-alert` for the animation and the HTML. You might make small changes, such as replacing the content of the `message-container` to use a for-loop instead.

Reply:
The idea is that they both represent the same concept, so they should be visual copycat as much as possible. We can't risk having different visual behavior here. The only reason we went with a new component approach is that we couldn't reuse restrictions alert to display messages because of the order of events.

Reply:
Literally do a copy and paste of the `restrictions-alert`? HTML, style and animations?

In case of the animations, it would require us to add the animations on parent of this component, just like it is done now with `restrictions-alert`